config/application.rb

```diff
@@ -8,9 +8,15 @@
 
 module Lobsters
   class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 5.0
```

Comment:
actually, I changed this back to 5.0 because that's what I realized `config/initializers/new_framework_defaults_5_2.rb` was for. Once all of those defaults are turned on, we can then remove that file and change this line to 5.2... man rails is tedious to upgrade